rocAL_pybind/amd/rocal/decoders.py

```diff
@@ -53,9 +53,9 @@ def image(*inputs, user_feature_key_map=None, path='', file_root='', annotations
     """
     reader = Pipeline._current_pipeline._reader
     Pipeline._current_pipeline._last_batch_policy = last_batch_policy
-    if last_batch_padded is False:
-        print("last_batch_padded = False is not implemented in rocAL. Setting last_batch_padded to True")
-        last_batch_padded = True
+    if pad_last_batch_repeated is False:
+        print("pad_last_batch_repeated = False is not implemented in rocAL. Setting pad_last_batch_repeated to True")
```

Comment:
Are we not implementing this? I thought this is handled?

Reply:
This is right now added only for the audio reader and image reader. Adding this statement for other readers. That should be enabled in the next PR